lonestar/analytics/distributed/tc/README.md

```diff
@@ -16,28 +16,32 @@ INPUT
 --------------------------------------------------------------------------------
 
 Takes in symmetric Galois .gr graphs that have been cleaned.
+You must specify the -symmetricGraph flag when running this benchmark.
 
 BUILD
 --------------------------------------------------------------------------------
 
 1. Run cmake at BUILD directory (refer to top-level README for cmake instructions).
-The DIST HETERO option to enable GPUs must be on. (see README in lonestardist)
 
-2. Run `cd <BUILD>/lonestardist/tc; make -j
+2. Run `cd <BUILD>/lonestar/analytics/distributed/triangle-counting; make -j
 
 RUN
 --------------------------------------------------------------------------------
 
 To run on 1 with a single GPU, use the following:
-`./tc <symmetric-input-graph> -pset=g -num_nodes=1`
+`./triangle-counting-dist <symmetric-input-graph> -symmetricGraph -pset=g -num_nodes=1`
+
+To run on 1 with a single CPU, use the following:
+`./triangle-counting-dist <symmetric-input-graph> -symmetricGraph -pset=c -num_nodes=1`
```

Comment:
pset and num_nodes are only required if GPU execution is enabled and is incorrect otherwise. I would note that down somewhere. Also, add a thread argument.